packages/babel-cli/src/babel/file.js

```diff
@@ -127,16 +127,10 @@ export default async function ({
   async function stdin(): Promise<void> {
     const code = await readStdin();
 
-    const res = await util.transform(
-      cliOptions.filename,
-      code,
-      defaults(
-        {
-          sourceFileName: "stdin",
-        },
-        babelOptions,
-      ),
-    );
+    const res = await util.transform(cliOptions.filename, code, {
+      sourceFileName: "stdin",
```

Comment:
@existentialism I guess this is a simpler example here.  We always want `stdin` to be applied here, I suppose?

If that's the case, then yes these are definitely out-of-order.